### PR TITLE
filter out left/right hands from joints list in avatarapp

### PR DESCRIPTION
**Pull request by ElderOrb**
should fix:
	FB17884 - Wearable disappears when added to some joints
	FB17885 - Changing werable joint moves the model and detaches from avatar body

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12476/HighFidelity-Beta-PR13897-dddfe85.apk

File: interface/resources/qml/hifi/avatarapp/AdjustWearables.qml [hunk 1]
```diff
@@ -25,7 +25,17 @@ Rectangle {
         modified = false;
     }
 
-    property var jointNames;
+    property var jointNames: []
+    onJointNamesChanged: {
+        jointsModel.clear();
+        for (var i = 0; i < jointNames.length; ++i) {
+            var jointName = jointNames[i];
+            if(jointName !== 'LeftHand' && jointName !== 'RightHand') {
```
Comment (ctrlaltdavid) [inline interface/resources/qml/hifi/avatarapp/AdjustWearables.qml]: Space missing between "if" and "(". Plus some similar occurrences elsewhere for "if" and "for".

Comment (ElderOrb) [reply interface/resources/qml/hifi/avatarapp/AdjustWearables.qml]: ooops, I did it again (c).... I beg pardon, is there any way to make QtCreator (which I am typically using as editor) to enforce hifi coding standards?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12514/HighFidelity-Beta-PR13897-2b34481.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12514/HighFidelity-Beta-Interface-PR13897-2b34481.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12514/HighFidelity-Beta-PR13897-2b34481.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12514/HighFidelity-Beta-Interface-PR13897-2b34481.exe

**Comment by ElderOrb** [comment]
PR updated to address regression: https://highfidelity.manuscript.com/f/cases/17981/After-PR13898-joints-list-in-Avatar-app-s-Joint-dropdown-menu-do-not-match-actual-joints

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12613/HighFidelity-Beta-PR13897-0773945.apk

**Comment by qliacoombs** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12628/HighFidelity-Beta-PR13897-0773945.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12628/HighFidelity-Beta-PR13897-0773945.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12628/HighFidelity-Beta-Interface-PR13897-0773945.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12628/HighFidelity-Beta-PR13897-0773945.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12628/HighFidelity-Beta-Interface-PR13897-0773945.exe

**Comment by qlimguicheteau** [comment]
Now Testing

**Comment by qlimguicheteau** [comment]
PASS @conklin94122 

Tested all joints on two avatars where effect was known to happen. Item stayed equipped.
Tested on a custom avatar with around 30 extra joints. No problems seen.

**Pull request merged**